bin/tblextr.py

```diff
@@ -544,6 +594,7 @@ def fill_ops_db(table_name, db, indir):
       dep_id += len(tid_list)
 
   if any_trace_found:
+    json_metadata_write(jsonfile, params, params2)
```

Comment:
Should be:
db.metadata_json(sysinfo_file)

Reply:
db object has no field for jsonfile, so I had to add it to the call db.metadata_json(jsonfile, sysinfo_file), other methods do the same in tblextr.py